platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -55,11 +55,21 @@ public static synchronized Mapbox getInstance(@NonNull Context context, @NonNull
    * @return Mapbox Access Token
    */
   public static String getAccessToken() {
+    validateMapbox();
     validateAccessToken();
     return INSTANCE.accessToken;
   }
 
   /**
+   * Runtime validation of Mapbox creation.
+   */
+  private static void validateMapbox() throws InvalidAccessTokenException {
+    if (INSTANCE == null) {
+      throw new InvalidAccessTokenException();
```

Comment:
The shown message to the user is:

>Using MapView requires setting a valid access token. Use `Mapbox.getInstance(Context context, String accessToken)` to provide one. Please see https://www.mapbox.com/help/create-api-access-token/ to learn how to create one. More information in this guide https://www.mapbox.com/help/first-steps-android-sdk/#access-tokens. 

INSTANCE being null matches behaviour of not having provided an accesstoken. I think we can get away with the same exception as both they mean the same thing. Do you feel that specific exception would provide more context than the message above?